lua_examples/pcm/play_file.lua

```diff
@@ -0,0 +1,40 @@
+-- ****************************************************************************
+-- Play file with pcm module.
+--
+-- Upload jump_8k.u8 to spiffs before running this script.
+--
+-- ****************************************************************************
+
+
+function cb_drained(d)
+  print("drained "..node.heap())
+
+  file.seek("set", 0)
+  -- uncomment the following line for continuous playback
+  --d:play(pcm.RATE_8K)
+end
+
+function cb_stopped(d)
+  print("playback stopped")
+  file.seek("set", 0)
+end
+
+function cb_paused(d)
+  print("playback paused")
+end
+
+file.open("jump_8k.u8", "r")
+
+drv = pcm.new(pcm.SD, 1)
```

Comment:
@devsaurus I've been working with @phando on the firmware and I can confirm that this was based on an older build of your pcm branch-- at this point it's about 6 weeks old.  latest commit in that build was `faab1ac devsaurus: implement current API with callbacks etc. -   (6 weeks ago)`.

@Phando told me it wasn't working and asked if I could include sigma-delta in the build.  I checked and saw that you made further changes and opened this PR so I rebuilt the firmware with your latest version.  I was in a rush and choose to go "rather safe than sorry" and included sigma-delta in the build, but I believe that it's not actually required for this.